addons/resource.language.en_gb/resources/strings.po

```diff
@@ -4979,7 +4979,11 @@ msgctxt "#10022"
 msgid "Settings - Games"
 msgstr ""
 
-#empty string with id 10023
+#. Dialog label used in add-on setup about user inform by remove of last configuration
```

Comment:
Dialog label used in add-on setup dialog to inform the user about removal of last add-on configuration